### DISCUSSION TITLE
Error: Element not found (Edge)

### DISCUSSION THREAD

**Discussion by JDittmerCOM** [apexcharts/apexcharts.js#3112] [Q&A]
Why are z. E.g. Edge doesn't show the charts?
I always get the message in the console that the element could not be found.

In Opera and Chrome e.g. B. the diagrams are displayed without problems.

Does anyone have any idea what could be causing this?

`Uncaught (in promise) Error: Element not found
    at apexcharts.min.js:14:37201
    at new Promise (<anonymous>)
    at t.value (apexcharts.min.js:14:21629)
    at statistiken:352:24`

**Comment by jonmes** [comment]
Just for the record If you are using Nuxt 3, It might be due to the server-side rendering (SSR) nature of Nuxt.

so make sure to name your apexcharts plugin as "apexcharts.client.js"
`import VueApexCharts from 'vue3-apexcharts'
export default defineNuxtPlugin((nuxtApp) => {
  nuxtApp.vueApp.use(VueApexCharts)
})`

and wrap your chart with <ClientOnly>

` <ClientOnly>
          <apexchart
                type="line"
                height="500"
                :options="chartOptions"
                :series="series"
               ref="chart"
            >
     </apexchart>
    </ClientOnly>`
